### DISCUSSION TITLE
Is there a simple way to change URL after user clicks a button?

### DISCUSSION THREAD

**Discussion by liamlawless35** [django-commons/django-unicorn#309] [Q&A] [answered]
Title. I want to make it so that the url is changed when the user clicks a button. Does anyone know if there's a simple way to do this? Is there some way to override unicorn's Javascript to add a small custom method for changing urls? (In case you can't tell I have no idea what I'm doing)

**Comment by adamghill** [comment, marked as answer]
You can use https://www.django-unicorn.com/docs/redirecting/#locationupdate to update the URL without actually redirecting the user. Let me know if that doesn't work for you!

**Reply by liamlawless35** [reply]
How in the name of all that is holy did I miss this!?? Thanks! I'll get back to you!
